config/spring19_seminar/cola_var/cola_bert.conf

```diff
@@ -0,0 +1,38 @@
+// Import the defaults using the relative path
+include "../final.conf"
```

Comment:
This has "max_epochs = -1"; are we doing this?

Reply:
Good catch. We should probably stick to a 3-epoch limit during both intermediate tuning and target task fine-tuning.

Reply:
fixed